src/Components/Onboarding/Steps/Artists/index.tsx

```diff
@@ -28,7 +28,7 @@ interface State {
 }
 
 export default class Artists extends React.Component<StepProps, State> {
-  static slug = "artists"
+  static slug: "artists" = "artists"
```

Comment:
Is specifying `"artists"` as a type necessary to make TS happy?

Reply:
Yeah, this is necessary to tell the type that we want a string value of "x". Otherwise, `typeof Artists.slug` will just infer that it is a String and not the actual value.